### ISSUE TITLE
Is batch attack supported for PGD?

### ISSUE THREAD

**Issue by minkyu-choi04** [BorealisAI/advertorch#92]
delta.requires_grad_()
    for ii in range(nb_iter):
        outputs = predict(xvar + delta)
        loss = loss_fn(outputs, yvar)
        if minimize:
            loss = -loss

        loss.backward()

This is part of the `advertorch/advertorch/attacks/iterative_projected_gradient.py `. 
I wonder if this PGD attack supports batch attack. I mean, can this function get a batch of different images and generates the corresponding attacks for each images? Because of `loss = loss_fn(outputs, yvar)`, it seems the batch is not supported. I just want to make sure I am right.
